src/components/button.js

```diff
@@ -0,0 +1,5 @@
+function Button(textContent) {
+
+	return `<button class="js-buttons">${textContent}</button>`;
```

Comment:
ok, now how do you do to make this return 

`button class="js-buttons"`, `button class="jjs-btn-iniciais js-btn-AC"`, `button class="js-btn-iniciais js-btn-ma-me"`, and so on?

you should write the test first!